examples/scripts/osft_gpt_oss_example.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+"""
+OSFT Training Example: GPT-OSS 20B
+
+This script demonstrates OSFT (Orthogonal Subspace Fine-Tuning) training with GPT-OSS 20B model
+using a single-node, multi-GPU setup with training_hub.
+
+OSFT allows continual training without catastrophic forgetting, making it ideal for:
+- Adapting GPT-OSS 20B to specialized domains (medical, legal, technical)
+- Adding new knowledge without degrading general capabilities
+- Fine-tuning without complex replay mechanisms
+
+Example usage:
+    python osft_gpt_oss_example.py \
+        --data-path /path/to/data.jsonl \
+        --ckpt-output-dir /path/to/checkpoints
+"""
+
+import os
+import sys
+import time
+from datetime import datetime
+import argparse
+import glob
+
+from training_hub import osft
+
+def find_most_recent_checkpoint(output_dir):
+    """
+    Find the most recent checkpoint in the training output directory.
+
+    Args:
+        output_dir (str): Training output directory containing hf_format/ subdirectory
+
+    Returns:
+        str: Path to the most recent checkpoint
+
+    Raises:
+        ValueError: If no checkpoints are found
+    """
+    # Get all checkpoint directories under hf_format
+    checkpoint_pattern = os.path.join(output_dir, "hf_format", "samples_*.0")
```

Comment:
This is technically correct, but we'll eventually want to change this in mini trainer since there's no reason for num samples to be a float. Can you please change this to just glob past the underscore?

```suggestion
    checkpoint_pattern = os.path.join(output_dir, "hf_format", "samples_*")
```